test/e2e/next-test-lib/next-modes/next-deploy.ts

```diff
@@ -89,6 +102,37 @@ export class NextDeployInstance extends NextInstance {
     // no-op as the deployment is created during setup()
   }
 
+  public async destroy(): Promise<void> {
+    if (this.isDestroyed) {
+      throw new Error(`next instance already destroyed`)
+    }
+
+    // During setup() the test site is deployed to Netlify
+    // Once testing is complete, we should delete the deploy again
+
+    if (!process.env.NEXT_TEST_SKIP_CLEANUP) {
+      console.log(`Deleting project with deploy_id ${this._deployId}`)
+
+      const deleteRes = await execa('ntl', ['api', 'deleteDeploy', '--data', `{ "deploy_id": "${this._deployId}" }`])
+  
+      if (deleteRes.exitCode !== 0) {
```

Comment:
[dust] Rename to `deleteResponse`.

```suggestion
      const deleteResponse = await execa('ntl', ['api', 'deleteDeploy', '--data', `{ "deploy_id": "${this._deployId}" }`])
  
      if (deleteResponse.exitCode !== 0) {
```